packages/cli/src/constructs/browser-check.ts

```diff
@@ -111,8 +111,12 @@ export class BrowserCheck extends Check {
 
     const deps: CheckDependency[] = []
     for (const { filePath, content } of parsed.dependencies) {
+      const relativePath = pathToPosix(path.relative(Session.basePath!, filePath))
```

Comment:
What happens if the main browser check script or setup/teardown scripts are outside of the `Session.basePath`? I'm not sure if checkly-runners supports that, or if we should add an error in that case as well.